src/account/multiple.js

```diff
@@ -134,17 +135,17 @@ export default AccountBase.compose(AsyncInit, {
      */
     _resolveAccount (account) {
       if (account === null) {
-        throw new Error('No account or wallet configured')
+        throw new UnknownAccountError()
       } else {
         switch (typeof account) {
           case 'string':
             decode(account, 'ak')
-            if (!this.accounts[account]) throw new Error(`Account for ${account} not available`)
+            if (!this.accounts[account]) throw new UnAvailableAccountError(account)
             return this.accounts[account]
           case 'object':
             return isAccountBase(account) ? account : MemoryAccount({ keypair: account })
           default:
-            throw new Error(`Unknown account type: ${typeof account} (account: ${account})`)
+            throw new UnknownAccountTypeError(account)
```

Comment:
it looks like a development-only error, maybe we can use a more general error class:
```suggestion
            throw new TypeError(`Account should be an address (ak-prefixed string), keypair, or instance of account base, got ${account} instead`)
```